portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.ts

```diff
@@ -0,0 +1,200 @@
+  PaymentOfficerTeam,
+  ReportPeriod,
+  getCurrentReportPeriodForBankSchedule,
+  getOneIndexedMonth,
+  isEqualMonthAndYear,
+} from '@ukef/dtfs2-common';
+import externalApi from '../../external-api/api';
+import api from '../../v1/api';
+import { getBusinessDayOfMonth } from '../../utils/date';
+import { isValidEmail } from '../../utils/string';
+import { BANK_HOLIDAY_REGION } from '../../constants/bank-holiday-region';
+import { BankResponse } from '../../v1/api-response-types';
+
+export type SendEmailCallback = (emailAddress: string, recipient: string, formattedReportPeriod: string) => Promise<void>;
+
+const DEFAULT_PAYMENT_OFFICER_TEAM_NAME = 'Team';
+
+/**
+ * Returns the utilisation report due date for the current month
+ */
+export const getReportDueDate = async (): Promise<Date> => {
+  const bankHolidays = await externalApi.bankHolidays.getBankHolidayDatesForRegion(BANK_HOLIDAY_REGION.ENGLAND_AND_WALES);
+  const businessDay = Number.parseInt(process.env.UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH!, 10);
+  const dateInReportMonth = new Date();
+  return getBusinessDayOfMonth(dateInReportMonth, bankHolidays, businessDay);
+};
+
+/**
+ * Returns the utilisation report due date for the current month in 'd MMMM yyyy' format
+ */
+export const getFormattedReportDueDate = async (): Promise<string> => {
+  const reportDueDate = await getReportDueDate();
+  return format(reportDueDate, 'd MMMM yyyy');
+};
+
+/**
+ * Returns the utilisation report chaser date for the current month - i.e. the date that a follow-up email should be
+ * sent to the bank to chase a report if not received by the due date
+ */
+export const getReportOverdueChaserDate = async (): Promise<Date> => {
+  const bankHolidays = await externalApi.bankHolidays.getBankHolidayDatesForRegion(BANK_HOLIDAY_REGION.ENGLAND_AND_WALES);
+  const businessDay = Number.parseInt(process.env.UTILISATION_REPORT_OVERDUE_CHASER_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH!, 10);
+  const dateInReportMonth = new Date();
+  return getBusinessDayOfMonth(dateInReportMonth, bankHolidays, businessDay);
+};
+
+/**
+ * Returns the given month and year formatted in the 'MMMM yyyy' format.
+ */
+const formatMonthAndYear = (monthAndYear: MonthAndYear): string => format(new Date(`${monthAndYear.year}-${monthAndYear.month}`), 'MMMM yyyy');
+
+/**
+ * Returns the given month formatted in the 'MMMM' format.
+ */
+const formatMonth = (month: OneIndexedMonth): string => format(new Date(`2023-${month}`), 'MMMM');
+
+/**
+ * Returns the given report period as a string.
+ * For a single month this is in 'MMMM yyyy' format.
+ * For a period of more than one month where the start and end month and in the same year this is in the 'MMMM to MMMM yyyy' format.
+ * For a period of more than one month where the start and end months are in different years this is in the 'MMMM yyyy to MMMM yyyy' format.
+ */
+export const getFormattedReportPeriod = (reportPeriod: ReportPeriod): string => {
```

Comment:
This is commonised in #2985 so we need to remember to update this instance of it too (this will get merged first)